<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>410</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Customer ID:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="transactionButton">
            <property name="text">
             <string>Transaction list</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="prodListButton">
            <property name="text">
             <string>Product list</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="12" column="2">
             <widget class="QLineEdit" name="dateLine"/>
            </item>
            <item row="8" column="1">
             <widget class="QLabel" name="descriptionLab">
              <property name="text">
               <string>Description:</string>
              </property>
             </widget>
            </item>
            <item row="15" column="1">
             <widget class="QLabel" name="countryLab">
              <property name="text">
               <string>Country:</string>
              </property>
             </widget>
            </item>
            <item row="14" column="1">
             <widget class="QLabel" name="threeLabel">
              <property name="text">
               <string>Customer ID:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLineEdit" name="oneLine"/>
            </item>
            <item row="5" column="2">
             <widget class="QLineEdit" name="stockLine"/>
            </item>
            <item row="12" column="1">
             <widget class="QLabel" name="dateLab">
              <property name="text">
               <string>Invoice Date:</string>
              </property>
             </widget>
            </item>
            <item row="16" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="addRow">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deleteRow_2">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="9" column="1">
             <widget class="QLabel" name="twoLab">
              <property name="text">
               <string>Quantity:</string>
              </property>
             </widget>
            </item>
            <item row="13" column="1">
             <widget class="QLabel" name="priceLab">
              <property name="text">
               <string>Price:</string>
              </property>
             </widget>
            </item>
            <item row="14" column="2">
             <widget class="QLineEdit" name="threeLine"/>
            </item>
            <item row="13" column="2">
             <widget class="QLineEdit" name="priceLine"/>
            </item>
            <item row="15" column="2">
             <widget class="QLineEdit" name="countryLine"/>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="stockLab">
              <property name="text">
               <string>Stock Code:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="oneLab">
              <property name="text">
               <string>Invoice:</string>
              </property>
             </widget>
            </item>
            <item row="8" column="2">
             <widget class="QLineEdit" name="descriptionLine"/>
            </item>
            <item row="9" column="2">
             <widget class="QLineEdit" name="twoLine"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
